docs/multiple-pulses.ipynb

```diff
@@ -0,0 +1,347 @@
+    "            unit='deg',\n",
+    "        ),\n",
+    "        close=sc.array(\n",
+    "            dims=['cutout'],\n",
+    "            values=[95.2, 162.8, 216.1, 263.1, 310.5, 371.6],\n",
+    "            unit='deg',\n",
+    "        ),\n",
+    "        phase=62.40 * deg,\n",
+    "        distance=8.8 * meter,\n",
+    "        name=\"Frame-overlap 1\",\n",
+    "    ),\n",
+    "    tof.Chopper(\n",
+    "        frequency=28 * Hz,\n",
+    "        open=sc.array(\n",
+    "            dims=['cutout'],\n",
+    "            values=[98.0, 154.0, 206.8, 254.0, 299.0, 344.65],\n",
+    "            unit='deg',\n",
+    "        ),\n",
+    "        close=sc.array(\n",
+    "            dims=['cutout'],\n",
+    "            values=[134.6, 190.06, 237.01, 280.88, 323.56, 373.76],\n",
+    "            unit='deg',\n",
+    "        ),\n",
+    "        phase=12.27 * deg,\n",
+    "        distance=15.9 * meter,\n",
+    "        name=\"Frame-overlap 2\",\n",
+    "    ),\n",
+    "]\n",
+    "\n",
+    "detectors = [\n",
+    "    tof.Detector(distance=32.0 * meter, name='detector'),\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4c3b69b7-5ebe-4188-aa4b-2c6621ec8017",
+   "metadata": {},
+   "source": [
+    "## Results\n",
+    "\n",
+    "We combine the `source`, `choppers`, and `detectors` into our `model`,\n",
+    "and then use the `.run()` method to execute the ray-tracing simulation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "05c191a2-81ef-41d3-8f64-343f72d7555d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = tof.Model(source=source, choppers=choppers, detectors=detectors)\n",
+    "res = model.run()\n",
+    "res"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7eea136a-4c5f-420c-8fd5-1cdc64be939f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "res.plot(max_rays=5000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9b761887-12f7-49f1-bbdb-a607c2376e47",
+   "metadata": {},
+   "source": [
+    "The time-distance diagram reveals that a small number of long-wavelength neutrons from one pulse are polluting the counts detected by the detector for the next pulse (red lines).\n",
+    "\n",
+    "The overlap is also visible when plotting the data seen by the detector,\n",
+    "even though the number of polluting neutrons is very small (the tails of each pulse are almost flat)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "95df472c-bc86-41b7-87f3-a1b8455e8fbc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "res.detectors['detector'].tofs.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "77d9a554-3cfc-4f24-a5a6-8c5e171ffbef",
+   "metadata": {},
+   "source": [
+    "Nevertheless, to try and obtain as clean as possible of a detector signal,\n",
+    "we include an additional chopper in the beamline to remove pulse overlap:"
```

Comment:
Not sure what the "nevertheless" is referring to?

Reply:
It was because the number of polluting neutrons is small, but the sentence also works fine if I remove "nevertheless"